src/liballoc/tests/vec.rs

```diff
@@ -1473,3 +1473,61 @@ fn vec_macro_repeating_null_raw_fat_pointer() {
         vtable: *mut (),
     }
 }
+
+#[test]
+fn partialeq() {
+    let vec1 = vec![1, 2];
+    let vec2 = vec![1, 2, 3];
+    let slice1 = &[1, 2];
```

Comment:
This is not a slice; it's a reference to an array.

Reply:
Fixed! I now have a proper slice and a separate reference to an array.